src/app/locationForm/common/Form.jsx

```diff
@@ -15,8 +15,18 @@ class Form extends Component {
     }
   }
 
+  isEditing(props){
+    return props.value === null ||  
+      ( typeof props.value === 'object' ? 
```

Comment:
If we want to keep the logic of whether a form is empty standardized like this in the HOC, we'll probably need to add more cases to it. For instance, once Phones supports an array, a new case for arrays will need to be added here.

As mentioned in another comment, IMO it would be more flexible to keep the HOC simple, and let the specific forms determine whether a given form is empty or not (letting the HOC know through a "isFormEmpty" prop or whatnot), even if it means a line or two of duplication in some of the forms that really do only have 1 simple value.

Reply:
I think it would make sense to pass this logic into the Form HOC as a prop whose value is a function. I'll add this prop as soon as we require it.

Reply:
Hmm, I was thinking it would be possible to do it even without a function. Since e.g. LocationForm controls the mapStateToProps which gives the Form its props, it could just determine the value there and pass it, without Form needing to be passed any function.
And yeah, some default could make sense either way. Form could fall back to checking `value` if no `isEmpty` prop is passed, or something like that.